shared/types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mozilla/rally-shared-types",
-  "version": "0.9.32",
+  "version": "0.9.33",
```

Comment:
@thomik-corp I think the shared-types version has already reached 0.9.34. so this should be 0.9.35?